tools/functional-tester/etcd-agent/agent.go

```diff
@@ -80,18 +81,56 @@ func (a *Agent) stop() error {
 	if a.state != stateStarted {
 		return nil
 	}
-	err := a.cmd.Process.Kill()
+
+	err := closeCmd(a.cmd, syscall.SIGTERM)
```

Comment:
let's only do one signal since we know etcd should and will response to it. drop either sigterm or sigint please.